Hudl.Mjolnir.Tests/Bulkhead/BulkheadFactoryTests.cs

```diff
@@ -46,52 +45,50 @@ public void Construct_InitializesConfigGauge_GaugeFiresForOneBulkhead()
             mockMetricEvents.Verify(m => m.BulkheadGauge(key, "semaphore", expectedMaxConcurrent, It.IsAny<int>()), Times.Once);
         }
 
-        [Fact]
-        public void Construct_InitializesConfigGauge_GaugeFiresForMultipleBulkheads()
-        {
-            // Arrange
-
-            var key1 = AnyString;
-            var groupKey1 = GroupKey.Named(key1);
-
-            var key2 = AnyString;
-            var groupKey2 = GroupKey.Named(key2);
-
-            var expectedMaxConcurrent1 = AnyPositiveInt;
-            var expectedMaxConcurrent2 = AnyPositiveInt;
-            
-            var mockMetricEvents = new Mock<IMetricEvents>(MockBehavior.Strict);
-            mockMetricEvents.Setup(m => m.BulkheadGauge(groupKey1.Name, "semaphore", expectedMaxConcurrent1, It.IsAny<int>()));
-            mockMetricEvents.Setup(m => m.BulkheadGauge(groupKey2.Name, "semaphore", expectedMaxConcurrent2, It.IsAny<int>()));
-
-            var mockBulkheadConfig = new Mock<IBulkheadConfig>(MockBehavior.Strict);
-            mockBulkheadConfig.Setup(m => m.GetMaxConcurrent(groupKey1)).Returns(expectedMaxConcurrent1);
-            mockBulkheadConfig.Setup(m => m.AddChangeHandler(groupKey1, It.IsAny<Action<int>>()));
-            mockBulkheadConfig.Setup(m => m.GetMaxConcurrent(groupKey2)).Returns(expectedMaxConcurrent2);
-            mockBulkheadConfig.Setup(m => m.AddChangeHandler(groupKey2, It.IsAny<Action<int>>()));
-
-            var mockLogFactory = new Mock<IMjolnirLogFactory>(MockBehavior.Strict);
-            mockLogFactory.Setup(m => m.CreateLog(It.IsAny<Type>())).Returns(new DefaultMjolnirLog());
-
-            // Act + Assert
-
-            var factory = new BulkheadFactory(mockMetricEvents.Object, mockBulkheadConfig.Object, mockLogFactory.Object);
-
-            // Wait 2s - since we haven't yet created any bulkheads, we shouldn't have any events.
-
-            mockMetricEvents.Verify(m => m.BulkheadGauge(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<int>(), It.IsAny<int>()), Times.Never);
-
-            Thread.Sleep(TimeSpan.FromMilliseconds(1500));
-
-            // Add two bulkheads
-            factory.GetBulkhead(groupKey1);
-            factory.GetBulkhead(groupKey2);
-
-            Thread.Sleep(TimeSpan.FromMilliseconds(1500));
-            
-            mockMetricEvents.Verify(m => m.BulkheadGauge(key1, "semaphore", expectedMaxConcurrent1, It.IsAny<int>()), Times.Once);
-            mockMetricEvents.Verify(m => m.BulkheadGauge(key2, "semaphore", expectedMaxConcurrent2, It.IsAny<int>()), Times.Once);
-        }
+//        [Fact]
```

Comment:
Maybe leave a comment about why this is disabled?

Reply:
This is re-implemented in https://github.com/hudl/Mjolnir/pull/64